spec/factories/payment_responses.rb

```diff
@@ -0,0 +1,86 @@
+FactoryBot.define do
+  sequence :adyen_payments_response do
+    OpenStruct.new(
+      response: {
+        "additionalData" => {
+            "recurringProcessingModel" => "UnscheduledCardOnFile"
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)